project_national/national_upgrades.yml

```diff
@@ -112,6 +112,8 @@ upgrades:
       - option: Insulation Ceiling|R-60
         apply_logic:
           or:
+            - Insulation Ceiling|R-30
+            - Insulation Ceiling|R-19
             - Insulation Ceiling|R-13
             - Insulation Ceiling|R-7
             - Insulation Ceiling|Uninsulated
```

Comment:
This enough to get some of these?

Reply:
I think so. According to resstock estimation options_saturation.tsv, these options would cover 71% of units that could have ceiling insulation.

https://github.com/NREL/resstock-estimation/blob/develop/projects/project_national/options_saturations.csv